float32[] float32Vector
